orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/KeelService.kt

```diff
@@ -16,22 +16,13 @@
 
 package com.netflix.spinnaker.orca
 
-import com.netflix.spinnaker.orca.keel.model.UpsertIntentRequest
+import com.netflix.spinnaker.orca.keel.model.DeliveryConfig
 import retrofit.client.Response
 import retrofit.http.Body
-import retrofit.http.DELETE
-import retrofit.http.GET
+import retrofit.http.Header
 import retrofit.http.POST
-import retrofit.http.Path
-import retrofit.http.Query
 
 interface KeelService {
-  @GET("/v1/intents/")
-  fun getIntents(): Response
-
-  @POST("/v1/intents/")
-  fun upsertIntents(@Body upsertIntentRequest: UpsertIntentRequest): Response
-
-  @DELETE("/v1/intents/{id}")
-  fun deleteIntents(@Path("id") id: String, @Query("status") status: String?): Response
+  @POST("/delivery-configs/")
```

Comment:
I have a general question - what happens when there's an existing delivery config for an application? will it be replaced/ignored?

Reply:
It gets updated:
https://github.com/spinnaker/keel/blob/cb6c5f8c977a61e67ecdccc627d154b92c4cbb6d/keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt#L39-L44